ee/WFLY-14061_EAP7-1366_JavaEE-to-JakartaEE-refs.adoc

```diff
@@ -0,0 +1,58 @@
+= Replace JavaEE references with corresponding JakartaEE ones
+:author:            Alessio Soldano
+:email:             asoldano@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+As WildFly is JakartaEE 8 certified since version 18 (https://github.com/wildfly/certifications/blob/EE8/WildFly_18.0.0.Final/jakarta-full-platform.adoc) and JakartaEE 9 implementation is in progress, we want to properly position the project as a JakartaEE implementation more than a JavaEE one.
+
+== Issue Metadata
+
+=== Issue
+
+* {issue-base-url}/EAP7-1366 [EAP7-1366]
+* {issue-base-url}/WFLY-14061 [WFLY-14061]
```

Comment:
FYI this is a community project so the primary issue should be the community issue as the EAP7 issues can not be accessed externally.  Also we should only need the WFLY issue in the file name.